infrastructure/docker-compose.yml

```diff
@@ -6,15 +6,14 @@ services:
     image: owasp/modsecurity:nginx-alpine
     container_name: web_application_firewall
     ports:
-      - "7999:8080"
       - "8000:8000"
+      - "40:40"
       - "80:80"
-      - "30:30"
     expose:
       - "8000"
-      - "443"
       - "80"
       - "30"
```

Comment:
Can we remove port 30 as it should only be exposed on webportal right..... waf(40) to webportal(30)

Reply:
ah yes i forgot to do that sorry let me do so now - had re-addded during testing